Dockerfile

```diff
@@ -51,4 +51,8 @@ ENV PATH /opt/venv/bin:$PATH
 RUN ["chmod", "g+w", "."]
 # Run bootstrap logic
 RUN ["chmod", "+x", "./bootstrap.sh"]
+
+HEALTHCHECK --interval=30s --retries=3 --timeout=1s \
+CMD curl -f http://localhost:9090/health || exit 1
```

Comment:
linkding has an option for changing the server port:
https://github.com/sissbruecker/linkding/blob/43d52642a69625f0706eae9e2ae7f61f20c3a24a/docs/Options.md#L65-L69

I think we need to change this so that the env variable is respected.

Reply:
Aha good call, I missed that env var. Updated.